runtime/src/main/java/io/quarkiverse/bucket4j/runtime/RateLimiterRuntimeConfig.java

```diff
@@ -76,6 +77,24 @@ interface Bucket {
     @ConfigGroup
     interface Limit {
 
+        /**
+         * If true, the limit is greedy and the quota is not consumed until the period is over.
+         */
+        @WithDefault("true")
+        boolean greedy();
+
+        /**
+         * The duration of first refill, typically it should be a moment in the future
```

Comment:
Something is off with this comment. Not sure how to phrase it, but a duration cannot be a moment

Reply:
i change property and comment. I'll need some time for write the tests.